pkg/apis/pingcap/v1alpha1/pd_config.go

```diff
@@ -250,7 +246,7 @@ type PDScheduleConfig struct {
 	// MaxStoreDownTime is the max duration after which
 	// a store will be considered to be down if it hasn't reported heartbeats.
 	// +optional
-	MaxStoreDownTime Duration `toml:"max-store-down-time,omitempty" json:"max-store-down-time,omitempty"`
+	MaxStoreDownTime string `toml:"max-store-down-time,omitempty" json:"max-store-down-time,omitempty"`
```

Comment:
`BurntSushi/toml` do not respect `MarshalText`, use `string` instead

Reply:
`BurntSushi/toml` seems not maintained anymore...

we can use `https://github.com/pelletier/go-toml` and implement `toml.MarshalTOML()` (like MarshalJSON())

Reply:
https://play.golang.org/p/_SfokqsaCPo it works

Reply:
Yes, that's exactly what I've thought.

Actually I've tried `go-toml`( and `naoina/toml`), it turns out that `go-toml` do not support `interface{}`. Also, PD and TiDB uses `BurntSushi/toml` so there is little verifying work, seems like that `BurntSushi/toml` is not an ideal but the only choice